horace_core/sqw/@SQWDnDBase/private/smooth_dnd.m

```diff
@@ -0,0 +1,157 @@
+function dout = smooth_dnd(din, xunit, varargin)
+% Smooths a 1,2,3 or 4 dimensional dataset
```

Comment:
Here or at L.9 or both, it would be good to say - either the `data_` field of an sqw or the whole of a dnd type object (which ignores the underlying implementation but clarifies for the user what dataset means here).